nipoppy/extractors/fmriprep/FC_configs.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Let's make this file a template i.e. `sample_FC_configs.json` so people can modify it without having it overwritten with future GH pull.

Reply:
@nikhil153  you mean just rename it to `sample_FC_configs.json`?